test/spec/properties.js

```diff
@@ -455,4 +467,140 @@ describe('properties', function() {
 
   });
 
+  describe('multiple inherited properties', function() {
+
+    // const createModel = createModelBuilder('test/fixtures/model/');
+    var mhModel = createModel([
+      'properties',
+      'multiple-inherited-properties'
+    ]);
+
+    describe('descriptor', function() {
+
+      it('should provide type', function() {
+
+        // when
+        var Type = mhModel.getType('mh:MultipleInherited');
+        var descriptor = model.getElementDescriptor(Type);
+
+        // then
+        expect(Type).to.exist;
+        expect(descriptor).to.exist;
+        expect(descriptor.propertiesByName.any).to.exist;
+        expect(descriptor.propertiesByName['mh:any']).to.exist;
+        expect(descriptor.propertiesByName['props:any']).to.exist;
+      });
+
+    });
+
+    describe('instance', function() {
+
+      it('should create instance', function() {
+
+        // when
+        var instance = mhModel.create('mh:MultipleInherited');
+
+        // then
+        expect(instance).to.exist;
+      });
+
+      describe('get', function() {
+
+        it('access via original name', function() {
+
+          // given
+          var instance = mhModel.create('mh:MultipleInherited');
+
+          // when
+          var property = instance.get('any');
+
+          // then
+          expect(property).to.exist;
```

Comment:
Verify actual property value, i.e. `expect(property).to.eql([])`.

TODO: Verify actual property value on non-many fields, too. Include test on default values, too.

Reply:
> Verify actual property value, i.e. expect(property).to.eql([]).

Good point. I keep one test for existance to fail early in case property is undefined.

> TODO: Verify actual property value on non-many fields, too.

Added,

Reply:
> Include test on default values, too.

You nailed it with this. [Added](https://github.com/bpmn-io/moddle/pull/39/commits/c332a7eaaecd0ddefa6b184120f6ff9fc2069110) the test and it fails!

Fixed it [here](https://github.com/bpmn-io/moddle/pull/39/commits/34457263442fd98c38df1e2e954fa8425a0b4492).
Sorry for the CI failure. I overlooked the missing semicolon because I also see linting errors for `.to.exist`, as this is not a function. Maybe add *dirty-chai* module for *eslint*? Also [fixed](https://github.com/bpmn-io/moddle/pull/39/commits/1b5ce8c45816ab50c8d112970a7ca932b04fc980) this of course.